src/service/packet_router.proto

```diff
@@ -48,6 +48,17 @@ message packet_router_packet_up_v1 {
   bytes signature = 10;
 }
 
+message packet_router_packet_report_v1 {
+  bytes packet_hash = 1;
+  uint64 timestamp = 2;
+  uint32 oui = 3;
+  float signal_strength = 4;
+  float frequency = 5;
+  string datarate = 6;
```

Comment:
Use an actual data rate type

Reply:
I'm seeing data rate as type `string` elsewhere in the repo and no `datarate` type. We can create a new type. What would be the parameters? A string of numbers? Number of bits + units (i.e 50kbs)?

Reply:
Coordinate with @andymck here but I think we should have a proto definition as defined here: https://github.com/helium/proto/pull/154/files#diff-6c154ef0b4726a72a348393855c692f17dc215d4201ab43b2660bb38dac0b6b3R5-R12 to have actual types for Datarate

Reply:
@mikev might be able to help here as well

Reply:
I agree datarate should be an enum.  An enum only uses one byte on disk or over the wire.

Here are the current known values:

        'SF12BW125' -> {12, 125};
        'SF11BW125' -> {11, 125};
        'SF10BW125' -> {10, 125};
        'SF9BW125' -> {9, 125};
        'SF8BW125' -> {8, 125};
        'SF7BW125' -> {7, 125};
        'SF12BW250' -> {12, 250};
        'SF11BW250' -> {11, 250};
        'SF10BW250' -> {10, 250};
        'SF9BW250' -> {9, 250};
        'SF8BW250' -> {8, 250};
        'SF7BW250' -> {7, 250};
        'SF12BW500' -> {12, 500};
        'SF11BW500' -> {11, 500};
        'SF10BW500' -> {10, 500};
        'SF9BW500' -> {9, 500};
        'SF8BW500' -> {8, 500};
        'SF7BW500' -> {7, 500};
        'LRFHSS1BW137' -> {7, 125};
        'LRFHSS2BW137' -> {7, 125};
        'LRFHSS1BW336' -> {7, 125};
        'LRFHSS2BW336' -> {7, 125};
        'LRFHSS1BW1523' -> {7, 125};
        'LRFHSS2BW1523' -> {7, 125};
        'FSK50' -> {7, 125};
        'RFU' -> {7, 125};